schema/artist/index.js

```diff
@@ -159,20 +169,47 @@ const ArtistType = new GraphQLObjectType({
             },
           },
         }),
-        resolve: ({ blurb, id }, { format }) => {
+        resolve: ({ blurb, id }, { format, partner_id, partner_bio }) => {
+          let bio = null;
```

Comment:
Why is this `bio` defined in this scope if it only gets used inside the resolved promise below?